### PR TITLE
Information about default StorageClasses

### PR DESCRIPTION
**Pull request by tamilselvan1102**
Issue https://github.com/kubernetes/website/issues/44932

### CODE REVIEW

**Review by Princesso — COMMENTED** [review]
/assign @sftim

**Comment by tamilselvan1102** [comment]
preview page: https://deploy-preview-44942--kubernetes-io-main-staging.netlify.app/docs/concepts/storage/dynamic-provisioning/#defaulting-behavior

**Comment by sftim** [comment]
I'm happy for anyone to review this PR.
/unassign

**Review by Princesso — COMMENTED** [review]
/assign @reylejano

File: content/en/docs/concepts/storage/dynamic-provisioning.md [hunk 1]
```diff
@@ -119,9 +119,10 @@ When a default `StorageClass` exists in a cluster and a user creates a
 `DefaultStorageClass` admission controller automatically adds the
 `storageClassName` field pointing to the default storage class.
 
-Note that there can be at most one *default* storage class on a cluster, or
-a `PersistentVolumeClaim` without `storageClassName` explicitly specified cannot
-be created.
+Note that If you set the `storageclass.kubernetes.io/is-default-class`
```
Comment (stmcginnis) [inline content/en/docs/concepts/storage/dynamic-provisioning.md:122]:
```suggestion
Note that if you set the `storageclass.kubernetes.io/is-default-class`
```

**Review by stmcginnis — COMMENTED** [review]
One trivial issue, otherwise lgtm.

**Comment by tamilselvan1102** [comment]
> One trivial issue, otherwise lgtm.
committed the suggested change
Thank you

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm
/approve

**Pull request merged**